src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -200,26 +200,43 @@ configurationRegistry.registerConfiguration({
 			tags: ['experimental', 'onExp']
 		},
 		[PromptsConfig.CONFIG_KEY]: {
-			type: 'object',
+			type: 'boolean',
 			title: nls.localize(
-				'chat.promptFiles.config.title',
+				'chat.reusablePrompts.config.enabled.title',
 				"Prompt Files",
 			),
 			markdownDescription: nls.localize(
-				'chat.promptFiles.config.description',
+				'chat.reusablePrompts.config.enabled.description',
+				"Enable reusable prompt files (`*{0}`) in Chat, Edits, and Inline Chat sessions. [Learn More]({0}).",
```

Comment:
Is that supposed to be `{1}` in the second arg?

Reply:
Yes!

Reply:
Fixed.